packages/server/controllers/posts.js

```diff
@@ -0,0 +1,64 @@
+const Post = require('../models/Post')
+
+module.exports = {
+    
+    createPost: async (req, res) => {
+        try {
+            const post = await Post.create({
+                title: req.body.title,
+                body: req.body.body,
+                isDraft: req.body.isDraft,
+            })
```

Comment:
We can simplify a bit of this by doing; this also keeps in line with the way you're doing it in edit post.
```suggestion
            const { title, body, isDraft } = req.body
            const post = await Post.create({ title, body, isDraft })
```